code/obj/machinery/interdictor.dm

```diff
@@ -191,16 +231,26 @@
 //call this from an event to determine if sufficient power is remaining to complete an interdiction,
 //passing an amount in cell charge that is required to interdict the event.
 //returns 1 if interdiction was successful, 0 if power was insufficient
-//second arg skips immediate visual update (use if potential for very high amounts of individual calls)
-/obj/machinery/interdictor/proc/expend_interdict(var/stopcost,var/skipanim)
-	if (status & BROKEN || !src.canInterdict)
+
+//first arg specifies the cell charge required to successfully Do the Thing
```

Comment:
As a proc should this comment not be in codedoc format to make it more useful when someone is writing a proc call?

Reply:
Rekerfuffled in 6ac706f; comment code-doc-ified, and updated a few of the variable names for a bit more clarity.